### ISSUE TITLE
Accessibility [NVDA] A11y_MicrosoftGuides_Outline_Completion_Screen Reader: NVDA is not announcing the text written inside completion step.

### ISSUE THREAD

**Issue by fsteffi** [nvaccess/nvda#10053]
“Check out Accessibility Insights! -  Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details: 
Application Name: Microsoft Guides – PC Authoring

Windows Version: Windows10

Desktop Application

Application Version: 103.1907.25003.0

NVDA Version: 2018.3.2

Logon Details as below:

Username: guidesselfhost@M365x699413.onmicrosoft.com
Password: No*****3

Instance: Test Test MRWNorthGuides Helf-host

Steps to Reproduce:

1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.

2. Sign-In into application using correct credentials.

3. Select the instance and click on Continue.

4. Navigate to any existing guides under the recent tab.

5. Start screen reader.

6. Navigate to Completion step.

Actual Result:

When NVDA is on & navigated to Completion step, NVDA is only announcing Completion step but not the content inside it.

This issue is not observed with Narrator.

Expected Result:

NVDA should announce the content written inside the step.

Recommendations:

Refer below link which is repository of bug fixes code snippets:

https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

User Impact:

Screen reader users would not come to know about the information given inside step.

MAS Reference:

MAS 1.3.1: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by DrSooom** [comment]
@fsteffi: Please try to reproduce this with [NVDA 2019.2 RC2](https://ci.appveyor.com/api/buildjobs/qqofdhd6mlh8mq5u/artifacts/output/nvda_2019.2rc2.exe) (running via the Installer should be enough) and run NVDA before starting Microsoft Guides – PC Authoring. Thanks.

**Comment by fsteffi** [comment]
@DrSooom I have done exactly as you have asked , but the bug still reproduces with  NVDA 2019.2 RC2.

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
